### DISCUSSION TITLE
title-block-banner displaying in preview but not when deployed?

### DISCUSSION THREAD

**Discussion by maxdrohde** [quarto-dev/quarto-cli#770] [Q&A] [answered]
I migrated my `distill` blog to `quarto` today, and I'm noticing a strange issue. When I view the website locally using `quarto preview` from the command line, I see the correct `title-block-banner`. However, when I view the website as deployed on Netlify, the color is wrong (it's all blue). See below for example.

## Correct (using `quarto preview` locally)
<img width="2560" alt="Screen Shot 2022-04-29 at 18 56 11" src="https://user-images.githubusercontent.com/25734232/166081967-be8aa016-c7ef-4212-ae5e-cf1cd9cf17a9.png">

## Wrong (deployed on Netlify)
<img width="2560" alt="Screen Shot 2022-04-29 at 18 56 24" src="https://user-images.githubusercontent.com/25734232/166081993-a0459c74-4802-4f03-9b71-5c187be2aed7.png">

Any ideas what the issue may be?

The github repo for the source is here: https://github.com/maxdrohde/blog_quarto

and the deployed website is here: https://maximilianrohde.com/

**Comment by maxdrohde** [comment, marked as answer]
Never mind, updated to the latest version of Quarto and it works now!

**Reply by dragonstyle** [reply]
Glad we fixed it! There were definitely issues with global vs. per page CSS that potentially could cause this type of behavior that I fixed a week or two ago!
